index.html

```diff
@@ -100,18 +104,20 @@ <h3 class="article__title">How can I help?</h3>
                     Learn more
                   </a>
                 </p>
+              <!--two boxes below image-->
               </div>
             </div>
           </div>
         </div>
+       
 
-        <div class="sidebar">
+        <sidebar class="sidebar">
           <h2 class="heading-underline">Upcoming events</h2>
-
+        
           <div class="article">
```

Comment:
You can change this `div` to the `article` tag too

Reply:
Thanks Lucy, amended